src/applications/discharge-wizard/components/AnswerReview.jsx

```diff
@@ -28,39 +28,31 @@ const AnswerReview = ({ formValues, handleScrollTo }) => {
           most accurate information regarding your discharge situation.
         </p>
       </div>
-      <table className="usa-table-borderless">
```

Comment:
It did not seem like `table` was being used here in the appropriate way. A table should be used when [ref](https://design.va.gov/components/table#when-to-use-a-table):

- presenting data that can be compared across multiple categories.
- organizing precise numerical data, such as statistical data.

And this information, listing answer results with an edit link, did not fall into those use cases. So instead of replacing this with `va-table`, I've simplified it with standard markup and flexbox.